.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -2,7 +2,7 @@
 name: Report a bug
 about: Let us know about a bug.
 title: ""
-labels: ""
+labels: "bug"
```

Comment:
If we do accept the templates, please don't add the bug label. Almost everything is a bug and this just adds noise.